lib/puppet/util/pidlock.rb

```diff
@@ -71,6 +91,7 @@ def clear_if_stale
       @lockfile.unlock unless exe_path =~ /\\bin\\ruby.exe$/
     end
   end
-  private :clear_if_stale
+
+  private :clear_if_stale, :ps_argument_for_current_kernel
```

Comment:
I generally put the `private :<method>` keyword immediately following the method, otherwise it's easy to miss at the bottom. So long as you specify which method is private, it won't affect methods that come after.